src/Apps/Order/Routes/Payment/PaymentContent.tsx

```diff
@@ -248,19 +261,21 @@ const getAvailablePaymentMethods = (
   // when available, unshift ACH since it's the first option we want to offer
   if (availablePaymentMethods.includes("US_BANK_ACCOUNT")) {
     paymentMethods.unshift(
-      <BorderedRadio
+      <RadioWithLabel
```

Comment:
**question:** Will the credit card and wire transfer radio buttons still have an underline when they are hovered or selected?

Reply:
I asked this question before you explained how the fix works, and this makes sense now too.